Accel labyrinth

Brydar42 opened the pull request:
heres our progress over the class

accelerometer and UI started

Brydar42 committed:
basic Xcode

here

Commit:
Initial Commit

Brydar42 committed:
Merge branch 'AccelLabyrinth' of https://github.com/k-kruusi/GameBase into AccelLabyrinth

Commit:
Updated ReadMe

Brydar42 committed:
Merge branch 'AccelLabyrinth' of https://github.com/k-kruusi/GameBase into AccelLabyrinth

Commit:
ReadMe Update

Added Me and My partners Name

Commit:
Merge branch 'AccelLabyrinth' of https://github.com/k-kruusi/GameBase into AccelLabyrinth

Commit:
ReadMe editing

Brydar42 committed:
removed peroid and extra space

Brydar42 committed:
March 8 changes

Base code for Accelerometer and added Marya's sceen file

Brydar42 committed:
Fixed falling blocks

kevin-kruusi commented on AccelLabrinth/AccelLabrinth/GameViewController.swift:
```diff
@@ -0,0 +1,66 @@
+        
+        if let view = self.view as! SKView? {
+            // Load the SKScene from 'GameScene.sks'
+            if let scene = SKScene(fileNamed: "GameScene") {
```
Creating your game scene by making an instance of itself, altho the method you use will work, it doesn't know its a GameScene vs SKScene.

```
let scene = GameScene(size: view.frame.size)
```

This way you will have access to custom functions and variables unique to your GameScene.

kevin-kruusi commented on AccelLabrinth/AccelLabrinth/GameViewController.swift:
```diff
@@ -0,0 +1,66 @@
+    }
+    @objc func update() {
+        if let accelerometerData = motionManager.accelerometerData {
+            //prints out the latest information from the accelerometer
```
You'll probably want to make your GameScene a private member variable so you can communicate to it outside of the viewDidLoad() function. For when your passing it accelerometer data, or think about moving the MotionManager to your GameScene.

kevin-kruusi commented on AccelLabrinth/AccelLabrinth/GameViewController.swift:
```diff
@@ -0,0 +1,66 @@
+            }
+            //call to start up accelermeter
+            motionManager.startAccelerometerUpdates()
+            //sets the update intervals to 1 second so it can change fast
```
1 second updates will probably be too slow.

Brydar42 committed:
Marya's game view fix and chage to update intervals

the border will not fall down or rotat and changed it from seconds update interval to milliseconds update interval

Brydar42 committed:
Base code for object motion

commented out apply force due to ball not existing

Brydar42 committed:
starting to make ball

Brydar42 committed:
Ball simple implentation

Brydar42 committed:
Ball setup to funtion like old lady

nvm the .ds_store but the ball will move like the zombie conga old lady

Brydar42 committed:
Marya's Sprites and start of adding goal and ending of game

all the assets are Marya work as I begin to work on the victory contitions while Marya works on gameplay

Brydar42 committed:
added victory and .ds_store is ignored but still here

added victroy based on hit detection on ball and goal

Brydar42 committed:
buttons are now included

buttons that make the player go up are now in

Brydar42 committed:
comments
